tests/posix_close/open_close.c

```diff
@@ -0,0 +1,143 @@
+#include <assert.h>
+#include <fcntl.h>
+#include <pthread.h>
+#include <stdlib.h>
+#include <signal.h>
+#include <string.h>
+#include <unistd.h>
+#include <stdio.h>
+
+#include "posix_close.h"
+#include "warnp.h"
+
+#include "open_close.h"
+
+static int alarmed = 0;
+
+struct open_close {
+	const char * filename;
+	int numbytes;
+	volatile int * stop;
+};
+
+static void
+alarm_handler(int signo)
+{
+
+	(void)signo; /*	UNUSED */
+	alarmed++;
```

Comment:
I didn't know what to do with the signals with which I was trying to interrupt `posix_close()`, but I figured that counting them would at least show that they were happening.